src/main/Validator.ts

```diff
@@ -66,6 +66,7 @@ const configDataSchemaV1 = Joi.object<ConfigV1>({
     useSpellChecker: Joi.boolean().default(true),
     enableHardwareAcceleration: Joi.boolean().default(true),
     autostart: Joi.boolean().default(true),
+    hideOnStart: Joi.boolean().default(false),
```

Comment:
you don't need this on v1 as it is not part of that version of the config file.

Reply:
Fixed.